Loop/Managers/DoseMath.swift

```diff
@@ -38,13 +38,13 @@ extension InsulinCorrection {
     ///   - scheduledBasalRate: The scheduled basal rate at the time the correction is delivered
     ///   - maxBasalRate: The maximum allowed basal rate
     ///   - duration: The duration of the temporary basal
-    ///   - minimumProgrammableIncrementPerUnit: The smallest fraction of a unit supported in basal delivery
+    ///   - rateRounder: The smallest fraction of a unit supported in basal delivery
     /// - Returns: A temp basal recommendation
     fileprivate func asTempBasal(
         scheduledBasalRate: Double,
         maxBasalRate: Double,
         duration: TimeInterval,
-        minimumProgrammableIncrementPerUnit: Double
+        rateRounder: ((Double) -> Double)?
```

Comment:
[nit] `rateRounder` to `basalRateRounder` to be consistent with `scheduledBasalRate` and `maxBasalRate`.

Reply:
I think those are a bit redundant, since the method name already has temp basal in it.

Reply:
👍